### DISCUSSION TITLE
Many to one, label of field missing after select

### DISCUSSION THREAD

**Discussion by wendevlin** [directus/directus#2758] [Archive]
Hello,

Thank you for directus! :-)

My version:
Directus 8.7.2 installed on a Webhoster directly from git

My issue:

I have 4 collections:

1. houses = a list of uniqu names
2. plays = a list of uniqu names
3. play in a house:
  - many to one house
  - many to one play
  - role: text input
4. dates:
  - many to one: play in a house
  - date
  - time
  - canceled: boolean

My problem is at the 4th collection:
For the many to one field I use the dropdown template `{{house.name}} - {{play.name}} - {{role}}`
This works like it should als long as I have not saved.
If I save I just see `- - rolename`

I found out that (what makes sense) just the id is saved into the field.
But I want to show of course the names of the 2 ids.

How can I achieve this?

some screenshots:

Create a new item, works fine.
![image](https://user-images.githubusercontent.com/12148533/84985843-c980bc00-b13d-11ea-865c-d36d3d0447da.png)

but after save:
![image](https://user-images.githubusercontent.com/12148533/84985937-f3d27980-b13d-11ea-9e4a-247dc5b576e8.png)

play in a house field options:
![image](https://user-images.githubusercontent.com/12148533/84986119-4449d700-b13e-11ea-972c-a17828d0fda0.png)

**Comment by rijkvanzanten** [comment]
The display side of things doesn't have support for that level of deeply nested values yet

**Reply by wendevlin** [reply]
Thanks @rijkvanzanten 
Does it come with version 9?

**Reply by rijkvanzanten** [reply]
Yeah! I've reworked the browse pages to first check with the displays what data they need before the items are fetched. In v8, the rows are fetched with everything 2 levels deep.
